tests/itests/src/test/java/org/codice/ddf/admin/query/request/SourcesRequestHelper.java

```diff
@@ -0,0 +1,317 @@
+                });
+    }
+
+    public boolean deleteSource(SourceType sourceType, String pid) {
+        GraphQlHelper.GraphQLRequest graphQLRequest = requestFactory.createRequest();
+        String jsonPath = "";
+
+        PidField pidField = new PidField();
+        pidField.setValue(pid);
+
+        switch (sourceType) {
+        case CSW:
+            graphQLRequest.usingMutation("DeleteCswSource.graphql");
+            jsonPath = "data.deleteCswSource";
+            break;
+        case WFS:
+            graphQLRequest.usingMutation("DeleteWfsSource.graphql");
+            jsonPath = "data.deleteWfsSource";
+            break;
+        case OPEN_SEARCH:
+            graphQLRequest.usingMutation("DeleteOpenSearchSource.graphql");
+            jsonPath = "data.deleteOpenSearchSource";
+        }
+
+        graphQLRequest.addArgument(PidField.DEFAULT_FIELD_NAME, pidField.getValue());
+
+        ExtractableResponse response = doSend(graphQLRequest);
+        if (responseHasErrors(response)) {
+            return false;
+        }
+
+        return extractResponse(response, jsonPath);
+    }
+
+    public boolean updateSource(SourceType sourceType, Map<String, Object> configToUpdate) {
+        GraphQlHelper.GraphQLRequest graphQLRequest = requestFactory.createRequest();
+        String jsonPath = "";
+
+        switch (sourceType) {
+        case CSW:
+            graphQLRequest.usingMutation("UpdateCswSource.graphql")
+                    .addArgument(CswSourceConfigurationField.DEFAULT_FIELD_NAME, configToUpdate);
+            jsonPath = "data.updateCswSource";
+            break;
+        case WFS:
+            graphQLRequest.usingMutation("UpdateWfsSource.graphql")
+                    .addArgument(WfsSourceConfigurationField.DEFAULT_FIELD_NAME, configToUpdate);
+            jsonPath = "data.updateWfsSource";
+            break;
+        case OPEN_SEARCH:
+            graphQLRequest.usingMutation("UpdateOpenSearchSource.graphql")
+                    .addArgument(OpenSearchSourceConfigurationField.DEFAULT_FIELD_NAME,
+                            configToUpdate);
+            jsonPath = "data.updateOpenSearchSource";
+        }
+
+        ExtractableResponse response = doSend(graphQLRequest);
+        if (responseHasErrors(response)) {
+            return false;
+        }
+
+        return extractResponse(response, jsonPath);
+    }
+
+    private boolean responseHasErrors(ExtractableResponse response) {
+        return response != null && response.jsonPath()
+                .get("errors") != null;
+    }
+
+    private ExtractableResponse doSend(GraphQlHelper.GraphQLRequest request) {
+        return request.send()
+                .getResponse();
+    }
+
+    private <T> T extractResponse(ExtractableResponse response, String jsonPath) {
+        return response.jsonPath()
+                .get(jsonPath);
+    }
+
+    public WfsSourceConfigurationField createWfsArgs(String sourceName, String pid,
```

Comment:
I think we should leave all the test data in the test class so this can remain an independent utility class. Can we do it for all the methods below that contain test data?